platformio_esp32_envs.ini

```diff
@@ -11,7 +11,7 @@ lib_ignore                = ESP8266WiFi, ESP8266Ping, ESP8266WebServer, ESP8266H
 [esp32_common]
 extends                   = common, core_esp32_3_3_2
 platform                  = ${core_esp32_3_3_2.platform}
-lib_deps                  = td-er/ESPeasySerial @ 2.0.8, I2Cdevlib-Core, adafruit/Adafruit ILI9341 @ ^1.5.6, Adafruit GFX Library, LOLIN_EPD, adafruit/Adafruit BusIO @ ^1.10.0, VL53L0X @ 1.3.0, SparkFun VL53L1X 4m Laser Distance Sensor @ 1.2.9, td-er/SparkFun MAX1704x Fuel Gauge Arduino Library @ ^1.0.1, ArduinoOTA, ESP32HTTPUpdateServer, FrogmoreScd30, Multi Channel Relay Arduino Library, SparkFun ADXL345 Arduino Library, ITG3205, Adafruit_ST77xx, ShiftRegister74HC595_NonTemplate
```

Comment:
Why remove the version nr and the adafruit prefix?

Reply:
I pulled the library local to make changes to the initialization, as is the easiest way to add support for the extra display models. Don't expect that a) Adafruit is interested in these mods, and b) seeing their often delayed response to PR's, it would take considerable time before being available for general use, once accepted.

Reply:
Maybe we should relabel it by making our own version/fork of it.
This may lead to some random choice by PlatformIO for which version will be picked.

Reply:
Renamed library to have an 'ESPEasy' postfix.